frontend/src/components/dialogs/ActionButtonDialog.vue

```diff
@@ -16,23 +16,29 @@ limitations under the License.
 
 <template>
   <div v-if="canPatchShoots">
```

Comment:
I think it is strange to decide inside a component that it should not render something. This should be done outside even if it redundant. What makes sense inside the component would be to disable the action.

Reply:
Okay but this has nothing to do with this PR. This is logic related to the member roles / permissions. I would suggest to change this when we introduce the uam role, which I will be working on next.